indra/llui/llui.cpp

```diff
@@ -592,11 +592,13 @@ namespace LLInitParam
 	
 	void ParamValue<const LLFontGL*>::updateBlockFromValue(bool make_block_authoritative)
 	{
-		if (getValue())
+		// It is important to use getRawValue() instead of getValue() here
+		// because getValue() calls validateBlock() and updateValueFromBlock()
+		if (getRawValue())
```

Comment:
This is clearly wrong. if you have to avoid validation you are likely missing something.

Reply:
This breaks TOS font size:
![image](https://github.com/secondlife/viewer/assets/117672381/160ea6d3-0f0a-416b-9a5c-4679dd0d2d7e)

Reply:
> updateBlockFromValue() serves to copy Value to Block

My understanding is that block's values take priority over value and it is correct unless programmer specificaly chose to override value from xml.

Reply:
I don't think that some of these terms could have a priority
And when we use these two methods we shouldn't think of priority
When we explicitly call `updateBlockFromValue()` we expect that Value should be copied to Block and not vice versa

Reply:
Note how all other updateBlockFromValue are implemented - you are making an outliner. Besides this is what broke TOS and font test floaters.

Reply:
Does this affect other floaters?

Reply:
May be some of preview floaters, trackbal, may be some others, haven't checked.
In my opinion 'font test' floater sudenly not behaving as it should is enough of a reason to request a change.

Reply:
It does affect a lot more, check the W, N, E, S leters:
![image](https://github.com/secondlife/viewer/assets/117672381/49082a83-f715-46d4-a0f9-ace8b6f4780f)

Reply:
Right. "Font test" is named that way for a reason.